internal/pkg/client/client_test.go

```diff
@@ -23,11 +23,11 @@ func TestNewHttpClient(t *testing.T) {
 }
 
 func TestWithHostUrl(t *testing.T) {
-	orgClient, _, _ := getMockedClientAndLogs(t, false)
+	orgClient, httpTransport, _, _ := getMockedClientAndLogs(t, false)
 	hostClient := orgClient.WithHostUrl("https://foo.bar")
 
 	// Mocked response
-	httpmock.RegisterResponder("GET", `=~.+`, httpmock.NewStringResponder(200, `test`))
+	httpTransport.RegisterResponder("GET", `=~.+`, httpmock.NewStringResponder(200, `test`))
```

Comment:
Namiesto volania `RegisterResponder` globalne nad kniznicou `httpmock` sa to vola nad instaciou `httpTransport` (`*MockedTransport`).